app/controllers/follow_requests_controller.rb

```diff
@@ -1,9 +1,11 @@
 class FollowRequestsController < ApplicationController
-  before_action :set_follow_request, only: %i[ show edit update destroy ]
+  before_action :set_follow_request, only: %i[show edit update destroy]
+  before_action :authorize_follow_request, only: %i[show edit update destroy]
```

Comment:
Try to stay consistent. You are using before_action { authorize(...) } in your comments controller. Stay consistent throughout your repo